frontend/src/components/Form/EditMapForm.tsx

```
@@ -0,0 +1,51 @@
+import React from 'react';
+import BaseWrapper from '../common/BaseWrapper';
+import FormWrapper from './FormWrapper';
+import { BaseMap, Map } from '@/types';
```

Biome

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)